### PR TITLE
Updates to README post release v2.0.0

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
## Description
Updates to README file with v2.0.0 changes. 

## Category

- [ ] Bugfix
- [ ] Enhancement
- [ ] New Feature
- [ ] Configuration Change
- [ ] Versioning Change
- [ ] Unit Test(s)
- [x] Documentation

### CODE REVIEW

**Comment by arvindkrishnakumar-okta** [comment]
> IIRC, the protocol should be prefixed, and we should probably add the comment back in place as well
> https://github.com/okta/okta-sdk-java/blob/master/examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java

Fixed.

File: examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java [hunk 1]
```diff
@@ -62,18 +62,18 @@
 
     private void createClient() {
         Client client = Clients.builder()
-            .setOrgUrl("{yourOktaDomain}")
-            .setClientCredentials(new TokenClientCredentials("{apiToken}"))
```
Comment (bdemers) [inline examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java:66]: use `{apiToken}`
https://github.com/okta/okta-commons-java/blob/master/config-check/src/main/java/com/okta/commons/configcheck/ConfigurationValidator.java#L139

File: examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java [hunk 2]
```diff
@@ -62,18 +62,18 @@
...
             .build();
     }
 
     private void createOAuth2Client() {
         Client client = Clients.builder()
-            .setOrgUrl("{yourOktaDomain}")
+            .setOrgUrl("https://{yourOktaDomain}")  // e.g. https://dev-123456.okta.com
             .setAuthorizationMode(AuthorizationMode.PRIVATE_KEY)
-            .setClientId("{clientId}")
+            .setClientId("yourClientID")
```
Comment (bdemers) [inline examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java:74]: use `{clientId}`

File: examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java:66]: will do.

**Pull request merged**